packages/components/src/Enumeration/Items/Item/item.snapshot.test.js

```diff
@@ -62,11 +62,22 @@ describe('Item', () => {
 		};
 
 		// when
-		const wrapper = renderer.create(
-			<Item {...props} />
-		).toJSON();
+		const wrapper = renderer.create(<Item {...props} />).toJSON();
 
 		// then
 		expect(wrapper).toMatchSnapshot();
 	});
+
+	it('should display value if it is a string', () => {
+		const props = {
+			item: {
+				...item,
+				values: 'toto',
+			},
+		};
+
+		const wrapper = renderer.create(<Item {...props} />).toJSON();
```

Comment:
for this kind of test I prefer the use of shallow rendering + find (no snapshot please)